lib/provisioning.js

```diff
@@ -137,7 +141,7 @@ function detect(options, callback) {
 		function detectIssues() {
 			results.issues = [];
 
-			if (!results.provisioning.development.length || !valid.development) {
+			if (results.provisioning.development.length >0 && !valid.development) {
```

Comment:
Consistency with the whitespace.

```suggestion
			if (results.provisioning.development.length > 0 && !valid.development) {
```